data/assets/scene/digitaluniverse/sdss.asset

```diff
@@ -19,14 +19,13 @@ local Object = {
     Type = "RenderablePointCloud",
     Enabled = true,
     Opacity = 0.8,
-    File = speck .. "SDSSgals.speck",
+    File = speck .. "sdss.speck",
     Coloring = {
       FixedColor = { 0.8, 0.8, 1.0 },
       ColorMapping = {
         File = speck .. "SDSSgals.cmap",
         ParameterOptions = {
-          { Key = "proximity", Range = { 1.0, 50.0 } },
-          { Key = "redshift", Range = { 0.0, 0.075 } }
```

Comment:
What's the reason for removing the redshift coloring?

Reply:
It doesn't work. I don't understand the parameters for these color maps, and the color map for proximity does not naturally lend itself to redshift. Ideally, we'd have two color maps, one for each. Is that possible?

Reply:
Currently, it is not, unfortunately. But it is something we have discussed that we want.  

I'll make sure to create an issue for it